.github/workflows/package-json-alpha-check.yml

```diff
@@ -0,0 +1,39 @@
+name: Check for Alpha Techsee Packages
+
+on:
+  workflow_call:
+    inputs:
+      package-path:
+        description: 'Path to the package.json file'
+        required: false
+        default: './package.json'
```

Comment:
and if file doesn't exist? it will just pass?

also, it only support single package.json per repo, right? consider making it into array

Reply:
you missed these two comments

Reply:
if there is no package.json then you dont need to use this action in the repo
that's a good idea. once a requirement will come  we will optimize the action

Reply:
ok about the array, we can improve later, but then you'll need to support both single item and multiple :wink: 

regarding package.json not available - it will fail silently, just check and throw error to prevent this user mistake

Reply:
@royricht 
you will get this error if no package.json

grep: ./package.json: No such file or directory
Error: Invalid package.json format.

Reply:
about the array i think i can call this flow multiple times from the main flow without change the action for example

jo```
bs:
  check-alpha-in-root:
    uses: TechSeePublic/techsee-actions/actions/package-json-alpha-check.yml@branch-name
    with:
      package-path: './package.json'  # Checking for alpha packages in the root package.json

  check-alpha-in-subfolder:
    uses: TechSeePublic/techsee-actions/actions/package-json-alpha-check.yml@branch-name
    with:
      package-path: './subfolder/package.json'  # Checking for alpha packages in a subfolder

  check-alpha-in-another-folder:
    uses: TechSeePublic/techsee-actions/actions/package-json-alpha-check.yml@branch-name
    with:
      package-path: './another-folder/package.json'  # Another package.json in a different folder
```